### PR TITLE
613-feat: Update communication section

### LINKED ISSUE #613: [JS/FE En Course] Update the Communication section to include both Discord and Telegram
https://rs.school/courses/javascript
image
The "Communication" section on the RS School website currently lists Discord as the main communication channel. However, we are now using both Discord and Telegram for communication. Please update the text and links in this section to reflect that.

**Modify Text:**

===
*Communication
RS School uses two main communication channels:

[Discord](https://discord.com/invite/uW5cCHR): Join the Discord server to see the latest news and chat with students.
[Telegram](https://t.me/RSS_EN): You can also join the official Telegram channel for updates and discussions.

===

### PR DESCRIPTION
**Pull request by KristiBo**
## What type of PR is this? (select all that apply)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Related Tickets & Documents

- Related Issue #613 
- Closes #613 

## Screenshots, Recordings

Before:

before

After:

after

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## Release Notes

- **New Features**
	- Introduced a new Telegram chat link for the JavaScript English community.
	- Added course-specific content for the JavaScript course, including new subtitles and paragraphs.

- **Bug Fixes**
	- Updated test suite to enhance the selection method for the Discord link element.

- **Style**
	- Restructured SCSS styles for improved organization without changing existing styles.

### CODE REVIEW

File: src/widgets/communication/ui/communication.module.scss [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
+      gap: 40px;
+      align-items: flex-start;
+    }
   }
-}
 
-.communication-paragraph {
-  padding-top: 5px;
-}
+  .communication-subtitle,
```
Comment (ansivgit) [inline src/widgets/communication/ui/communication.module.scss:18]: Why do we need to overwrite styles for subtitle?

File: src/widgets/communication/ui/communication.tsx [hunk 2]
```diff
@@ -47,26 +54,48 @@ export const Communication = ({ courseName, lang = 'en' }: CommunicationProps) =
...
           <div>
-            <Subtitle>{subTitle}</Subtitle>
-            <Paragraph className={cx('communication-paragraph')}>
-              {firstParagraphFirstHalf}
-              <LinkCustom href={DISCORD_LINKS[courseName]} external>
-                {discordLink}
-              </LinkCustom>
-              {firstParagraphSecondHalf}
-            </Paragraph>
+            {courseName === JS_EN_COURSE
```
Comment (ansivgit) [inline src/widgets/communication/ui/communication.tsx:57]: Maybe we could combine this code?

File: src/widgets/communication/ui/communication.module.scss (hunk 1 above)
Comment (KristiBo) [reply src/widgets/communication/ui/communication.module.scss:18]: Previously padding-top was set for the paragraph but it's better to use padding-bottom. I changet it to padding-bottom for the subtitle and one paragraph.
With the default styles it looks worse in my opinion.
image

**Review by Quiddlee — APPROVED** [review]

File: src/widgets/communication/ui/communication.module.scss (hunk 1 above)
Comment (KristiBo) [reply src/widgets/communication/ui/communication.module.scss:18]: We discussed it with Anna and decided to leave it as is

File: src/widgets/communication/ui/communication.tsx [hunk 3]
```diff
@@ -47,26 +54,36 @@ export const Communication = ({ courseName, lang = 'en' }: CommunicationProps) =
             <Image src={discordLogo} alt="discord logo" />
           </figure>
           <div>
-            <Subtitle>{subTitle}</Subtitle>
-            <Paragraph className={cx('communication-paragraph')}>
-              {firstParagraphFirstHalf}
-              <LinkCustom href={DISCORD_LINKS[courseName]} external>
-                {discordLink}
+            <Subtitle className={cx('communication-subtitle')}>
+              {isJsEnCourse ? subTitleJs : subTitle}
```
Comment (Quiddlee) [inline src/widgets/communication/ui/communication.tsx:58]: please, move all the conditions above the `return` statement
```tsx
const courseSubTitle = isJsEnCourse ? subTitleJs : subTitle;
```

```suggestion
              {subTitle}
```

Comment (KristiBo) [reply src/widgets/communication/ui/communication.tsx]: Done

**Pull request merged**